src/components/select/filterable-select/filterable-select.spec.tsx

```diff
@@ -1137,9 +1132,9 @@ describe("FilterableSelect", () => {
       });
 
       it("should not reopen the SelectList when a user selects and Option by clicking", () => {
-        const container = document.createElement("div");
```

Comment:
**suggestion(non-blocking)**: what do you think about removing `enzymeContainer` attachment code here, given we have delated this to the setup/teardown hooks on lines 54 and 60?

Reply:
good spot that this isn't needed any more. I've removed it and cleaned it up, along with a few other instances of the same in the tests for all 3 select components - all tests still pass 👍